app/views/categories/_index_partial.html.haml

```diff
@@ -5,7 +5,7 @@
   %tr
     %th Name
     %th Depth
-    - if can_see? :destroy, @categories
+    - if can_see_header? [:destroy], @categories
```

Comment:
Here and the following cases, I changed can_see? method with another similar, it depends on the input parameters:
`can_see_any?`  -> for different actions for an item
`can_see_header?`  -> for different actions for different items